### PR TITLE
tests: kernel: Add description for test case

### PR DESCRIPTION
**Pull request by ajaykish**
Add description to test case in tests/kernel/gen_isr_table

Signed-off-by: Ajay Kishore <ajay.kishore@intel.com>

### CODE REVIEW

File: tests/kernel/gen_isr_table/src/main.c [hunk 1]
```diff
@@ -182,6 +182,30 @@ static int check_sw_isr(void *isr, u32_t arg, int offset)
 }
 #endif
 
+/**
+ * @defgroup kernel_interrupt_tests Interrupt
```
Comment (nashif) [inline tests/kernel/gen_isr_table/src/main.c]: this needs to only be defined one time, it is already defined in another PR.

**Review by nashif — CHANGES_REQUESTED** [review]

**Comment by ajaykish** [comment]
@nashif as you asked I removed the definition. Please have a look.

File: tests/kernel/gen_isr_table/src/main.c [hunk 2]
```diff
@@ -182,6 +182,23 @@ static int check_sw_isr(void *isr, u32_t arg, int offset)
 }
 #endif
 
+/** @brief test to validate gen_isr_table
+ *
```
Comment (spoorthik) [inline tests/kernel/gen_isr_table/src/main.c]: I think we should have  "@addtogroup kernel_interrupt_tests".

**Review by spoorthik — DISMISSED** [review]

File: tests/kernel/gen_isr_table/src/main.c (hunk 2 above)
Comment (nashif) [reply tests/kernel/gen_isr_table/src/main.c]: @ingroup rather.

**Comment by ManojSubbarao** [comment]
@ajaykish Please wrap this quickly addressing all review comments.

**Comment by ajaykish** [comment]
@nashif addressed your comment, please have a look.

**Review by nashif — APPROVED** [review]

**Pull request merged**